packages/vue-components/stories/OmegaForm/Union.vue

```diff
@@ -46,7 +46,8 @@ class B extends S.TaggedClass<B>()("B", {
 const schema = S
   .Struct({
     title: S.String,
-    union: S.Union(A, B)
+    // testing it works! union is nullable no asterisk showed
+    union: S.NullOr(S.Union(A, B))
```

Comment:
If it allows null id expect also to find a select entry representing null, so you can test and confirm the null case works as expected